src/db_adapters/assets/events.rs

```diff
@@ -0,0 +1,106 @@
+use crate::db_adapters::assets;
+use actix_diesel::{AsyncError, Database};
+use diesel::PgConnection;
+use tracing::warn;
+
+use super::event_types;
+
+pub(crate) async fn store_events(
+    pool: &Database<PgConnection>,
+    streamer_message: &near_indexer::StreamerMessage,
+) -> anyhow::Result<()> {
+    for shard in &streamer_message.shards {
```

Comment:
futures

Reply:
Just for the reference: https://github.com/near/near-indexer-for-explorer/pull/228